static/js/admin.js

```diff
@@ -287,6 +295,20 @@ function _setup_page() {
         error: failed_listing_streams
     });
 
+    // Populate realm aliases
+    channel.get({
+        url: "/json/realm/aliases",
+        idempotent: true,
+        timeout: 10*1000,
+        success: function (aliases_data) {
+            update_alias_label();
+            _.each(aliases_data.aliases, function (alias) {
+                add_alias(alias);
+            });
+        },
+        error: failed_listing_aliases
+    });
```

Comment:
You shouldn't need to do this `channel.get` query; the realm domains are available in `page_params.domains` and with a proper real-time sync implementation, the frontend will never need to do an extra AJAX request to the server to get this data.